### ISSUE TITLE
Does not find kernel headers on ubuntu-20.04 hwe

### ISSUE THREAD

**Issue by uli-heller** [tycho-kirchner/shournal#7]
```
( cd build && cmake -DSHOURNAL_EDITION=ko .. )
```

... does not find the kernel headers.
I've created a branch to fix the issue. See https://github.com/uli-heller/shournal/tree/ubuntu-2004-hwe for details.
Maybe I'll create a PR in case you're interested

**Comment by tycho-kirchner** [comment]
Thanks for reporting this issue and also for your pull request,  which I just merged. However, for the record, `kernel/cmake/FindKernelHeaders.cmake` succeeding in finding the kernel headers is not important for building the module - therefore it only emits a warning. The build is actually performed by the `kernel/Kbuild` makefile (called by `kernel/CMakeLists.txt`) which finds the headers on its own - if they are installed. So, in your case, installing `linux-headers-generic-hwe-20.04` beforehand is enough for the build to succeed, regardless of the fix of your PR. I accepted it anyway, because `FindKernelHeaders.cmake` does serve a purpose - it makes intellisense in my IDE (Qt Creator) work, and it does not harm to have it working on more systems 😉 (I'm running Debian). Further, I just added commit 32981f0f8c03a8f8460063aed86412c179bd499e which explicitly documents the kernel header dependency also for the DEB package. And I shortened the `README-compile` a bit.
Thanks
Tycho
